currentScript.js

```diff
@@ -22,7 +22,7 @@
 
           // For all scripts on the page, if src matches or if ready state is interactive, return the script tag
           for(i in scripts){
-            if(scripts[i].src == res || scripts[i].readyState == "interactive"){
+            if(res && scripts[i].src == res || scripts[i].readyState == "interactive"){
```

Comment:
I believe the correct fix here would be `scripts[i] && (scripts[i].src || scripts[i].readyState == "interactive")` If `scripts[i]` is undefined it will still error on the `.readyState` access no?

Reply:
yep